pkgs/top-level/all-packages.nix

```diff
@@ -18660,6 +18660,8 @@ in
     python3Packages = python36Packages;
   };
 
+  kvirc = callPackage ../applications/networking/irc/kvirc { };
```

Comment:
From section 9.15.2 of the nixpkgs manual:
> Call your application expression using libsForQt5.callPackage instead of callPackage. Import dependencies unqualified, i.e., qtbase not qt5.qtbase. Do not import a package set such as qt5 or libsForQt5.